https_www_redirects.conf

```diff
@@ -335,3 +335,52 @@ server {
       return         301 https://www.cpro.gov.sg$request_uri;
 }
 
+server {
+      listen         443 ssl http2;
+      server_name   cpro.gov.sg;
+      ssl_certificate /ssl/cpro.gov.sg.crt; 
+      ssl_certificate_key /ssl/cpro.gov.sg.key;
+      return         301 https://www.cpro.gov.sg$request_uri;
+}
+
+server {
+      listen         443 ssl http2;
+      server_name   cpro.gov.sg;
+      ssl_certificate /ssl/cpro.gov.sg.crt; 
+      ssl_certificate_key /ssl/cpro.gov.sg.key;
+      return         301 https://www.cpro.gov.sg$request_uri;
+}
+
+server {
+      listen         443 ssl http2;
+      server_name   cpro.gov.sg;
+      ssl_certificate /ssl/cpro.gov.sg.crt; 
+      ssl_certificate_key /ssl/cpro.gov.sg.key;
+      return         301 https://www.cpro.gov.sg$request_uri;
+}
```

Comment:
What's with the duo's?

Reply:
@lisatjide

Reply:
sorz, accidentally copied an additional set. i've removed it